src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -57,6 +57,22 @@ def __init__(
         self._seen_tables: dict[str, str] = {}
         self._principal_grants = principal_grants
 
+    def get_remaining_tables(self) -> list[Table]:
+        table_rows: list[Table] = []
+        for crawled_table in self._tc.snapshot():
+            if not self._is_migrated(crawled_table.database, crawled_table.name):
+                table_rows.append(crawled_table)
+                logger.info(f"remained-table-to-migrate: {crawled_table.key}")
```

Comment:
Still in doubt about the wording here: it might just be a "remaining" table, maybe it is not migrated at all. At least, it is a "non-migrated" table, but I prefer to avoid negation in wording. Maybe:

```suggestion
logger.info(f"remained-hive-metastore-table: {crawled_table.key}")
```

Reply:
Done